src/App.tsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react'
 import logo from './logo.svg'
 import './App.css'
-import Button from '@material-ui/core/Button'
+import { FileImportButton } from './Components/FileImportButton'
```

Comment:
I'd probably export the main component as the default export (you can still have non-default exports alongside if needed), such that these braces aren't needed.

Reply:
This I resolved this how you wanted